src/Components/CreateImageWizard/ValidatedInput.tsx

```diff
@@ -50,28 +51,34 @@ export const HookPasswordValidatedInput = ({
   warning = undefined,
   inputType,
   isDisabled,
+  isEmpty,
 }: HookValidatedInputPropTypes) => {
+  const validated = isEmpty
+    ? 'default'
+    : stepValidation.errors[fieldName] === 'default'
+    ? 'default'
+    : stepValidation.errors[fieldName]
+    ? 'error'
+    : 'success';
   const [isPasswordVisible, setIsPasswordVisible] = useState(false);
   const togglePasswordVisibility = () => {
     setIsPasswordVisible(!isPasswordVisible);
   };
 
   return (
     <>
+      {inputType === 'textInput'}
       <InputGroup>
         <InputGroupItem isFill>
-          <HookValidatedInput
+          <TextInput
```

Comment:
I think this should stay as `HookValidatedInput` so we can use the validation features we already have.

Reply:
the thing is that if I am using the new refactor component <ValidatedInputAndTextArea>
I have a problem that I mentioned before with the place of the error message, and I solved that with adding a condition to HookValidatedInput  - https://github.com/osbuild/image-builder-frontend/commit/93c677848d7b9fa81fa489b8d1d29ed180777ae2
but you dont like this solution 
<img width="1294" alt="Screenshot 2025-03-10 at 19 41 36" src="https://github.com/user-attachments/assets/738aada1-3c9e-4688-899c-11d38cf6d4f1" />